source/docs/advanced/payments/adyen.md

```diff
@@ -189,6 +210,79 @@ Add the following line to your `web/theme/modules/_modules.scss` file to load th
 @import "~theme/modules/Adyen/dropinCustomizations";
 ```
 
+### Add webhook
+
+From your adyen Customer Area under `Developers > Webhooks` click on `+ Webkook` button on the right top corner then click on `Add` for the `Standard Notification` type. Fill the fields as follows:
+
+- Transport:
+  - URL: the full url of your site followed by `/webhooks/payment/notification/adyen` e.g. `https://www.example.com/webhooks/payment/notification/adyen`
+  - Method: JSON
+- Authentication:
+  - User Name: the `FRONT_COMMERCE_ADYEN_NOTIFICATION_USERNAME` environement variable defined above
+  - Password: the `FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD` environement variable defined above
+- Additional Settings
+  - HMAC Key (HEX Encoded): click generate and copy it to `FRONT_COMMERCE_ADYEN_HMAC_KEY` environement variable defined above
```

Comment:
```suggestion
  - User Name: the `FRONT_COMMERCE_ADYEN_NOTIFICATION_USERNAME` environment variable defined above
  - Password: the `FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD` environment variable defined above
- Additional Settings
  - HMAC Key (HEX Encoded): click generate and copy it to `FRONT_COMMERCE_ADYEN_HMAC_KEY` environment variable defined above
```

(as said in a previous PR, you should install a spellchecker in your editor to avoid this kind of typos.)

Reply:
installed